docs/source/conf.py

```diff
@@ -195,6 +196,24 @@
     "torch": ("https://pytorch.org/docs/stable/", None),
 }
 
+# Redirects for moved pages
+redirects = {
+    "getting-started-setup": "getting-started.html",
+    "export-overview": "using-executorch-export.html",
+    "runtime-build-and-cross-compilation": "using-executorch-building-from-source.html",
+    "tutorials/export-to-executorch-tutorial": "../using-executorch-export.html",
+    "running-a-model-cpp-tutorial": "using-executorch-cpp.html",
+    "build-run-vulkan": "backends-vulkan.html",
+    "executorch-arm-delegate-tutorial": "backends-arm-ethos-u.html",
+    "build-run-coreml": "backends-coreml.html",
+    "build-run-mediatek-backend": "backends-mediatek.html",
+    "build-run-mps": "backends-mps.html",
+    "build-run-qualcomm-ai-engine-direct-backend": "backends-qualcomm.html",
+    "build-run-xtensa": "backends-cadence.html",
+    "apple-runtime": "using-executorch-ios.html",
```

Comment:
Not sure we have any of these left

Reply:
Yeah, the goal of this change is to make these old links point to the new pages. That way any external links (and SEO) will continue to work.